lib/webtrit_adapter_config.ex

```diff
@@ -49,6 +49,11 @@ defmodule WebtritAdapterConfig do
     Application.get_env(:webtrit_adapter, :portabilling_demo_i_custom_field)
   end
 
+  @spec portabilling_account_password_length :: integer() | nil
```

Comment:
After moving to assign a default value to the please of env variable read, this function will not return `nil`, and in general it is more clean regarding documentation that states about the default value to not return `nil`.